src/content/ja/2022/sustainability.md

```diff
@@ -0,0 +1,814 @@
+        <td class="numeric">92</td>
+      </tr>
+      <tr>
+        <td>Joomla</td>
+        <td>デスクトップ</td>
+        <td class="numeric">26</td>
+        <td class="numeric">452</td>
+        <td class="numeric">86</td>
+        <td class="numeric">1,690</td>
+        <td class="numeric">104</td>
+      </tr>
+      <tr>
+        <td>Joomla</td>
+        <td>モバイル</td>
+        <td class="numeric">22</td>
+        <td class="numeric">401</td>
+        <td class="numeric">83</td>
+        <td class="numeric">1,504</td>
+        <td class="numeric">82</td>
+      </tr>
+      <tr>
+        <td>Wix</td>
+        <td>デスクトップ</td>
+        <td class="numeric">123</td>
+        <td class="numeric">1,318</td>
+        <td class="numeric">86</td>
+        <td class="numeric">647</td>
+        <td class="numeric">197</td>
+      </tr>
+      <tr>
+        <td>Wix</td>
+        <td>モバイル</td>
+        <td class="numeric">118</td>
+        <td class="numeric">1,215</td>
+        <td class="numeric">9</td>
+        <td class="numeric">290</td>
+        <td class="numeric">148</td>
+      </tr>
+      <tr>
+        <td>Squarespace</td>
+        <td>デスクトップ</td>
+        <td class="numeric">27</td>
+        <td class="numeric">997</td>
+        <td class="numeric">89</td>
+        <td class="numeric">1,623</td>
+        <td class="numeric">214</td>
+      </tr>
+      <tr>
+        <td>Squarespace</td>
+        <td>モバイル</td>
+        <td class="numeric">27</td>
+        <td class="numeric">990</td>
+        <td class="numeric">89</td>
+        <td class="numeric">1,790</td>
+        <td class="numeric">202</td>
+      </tr>
+    </tbody>
+  </table>
+<figcaption>{{ figure_link(caption="CMS、デバイス、およびリソースタイプ別の中央値キロバイト", sheets_gid="1561070567", sql_file="cms_bytes_per_type.sql") }}</figcaption>
+</figure>
+
+{{ figure_markup(
+  image="median-kilobytes-by-cms-and-resource-type-desktop.png",
+  caption="デスクトップでのCMSおよびリソースタイプ別の中央値キロバイト",
+  description="",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=159460424&format=interactive",
+  sheets_gid="1561070567",
+  sql_file="cms_bytes_per_type.sql"
+) }}
+
+{{ figure_markup(
+  image="median-kilobytes-by-cms-and-resource-type-mobile.png",
+  caption="モバイルでのCMSおよびリソースタイプ別の中央値キロバイト",
+  description="",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQvn7rDUZ96mbcJGd-R-gGdofTptGuReAxtTp-jYGUPpXaDO11ef2LjXz_aj-bk7wIA3gvFbEX_El-e/pubchart?oid=1726838193&format=interactive",
+  sheets_gid="1561070567",
+  sql_file="cms_bytes_per_type.sql"
+) }}
+
+上の表およびグラフは、Wixがモバイルイメージの最適化により積極的に取り組んでいることを示しています。Next.jsやNuxt.jsのようなフレームワークを使用しているサイトジェネレーターのセグメントでも同様のパターンが見られます。
+
+この見かけ上小さな洞察は、プラットフォームおよびフレームワークがより持続可能なウェブサイトを提供するために果たす重要な役割を捉えています。適切なデフォルトを適用することにより、プラットフォーム開発者およびフレームワークの著者は、開発者がツールを活用して、_<a hreflang="en" href="https://screenspan.net/blog/green-by-default/">green by default</a>_のサイトを作成できるように支援できます。
```

Comment:
Does "green by default" need a translation or is it OK as is?

Reply:
Thank you for your question. The phrase "green by default" does not need to be translated